mautic/mautic
# [UI] Fix nav tabs wrap

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13826 



## Description

This pull request addresses an issue with navigation tabs that were previously stacking incorrectly due to overflow. The tabs now have a horizontal scroll feature, allowing users to navigate seamlessly without clutter. Custom buttons were implemented to wrap nav tabs and allow you to scroll with 1 click.




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/1bc2bc99-fd43-4cc9-b90e-c3611af25ff4)                                      | ![image](https://github.com/user-attachments/assets/a77d326b-87af-48b3-a866-3d4fdee1ced5)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Activate mobile view, open the Forms page, create a new form, and go to the fields tab. Add a checkbox group and ensure that the tabs are scrollable to the right. Verify that the scroll happens using the buttons.
